planning/route_handler/src/route_handler.cpp

```diff
@@ -1313,6 +1322,121 @@ lanelet::routing::RelationType RouteHandler::getRelation(
 
 lanelet::ConstLanelets RouteHandler::getShoulderLanelets() const { return shoulder_lanelets_; }
 
+std::array<double, 4> RouteHandler::getLaneletScope(
```

Comment:
This name and implementation highly depend on the drivable area computation algorithm. The function name and inputs should be more general, like `getLaneletWithDistance(lanes, nearest_idx, ego_pose, fwd_dist, bkd_dist)` so that we can omit the drivable area related parameters from the `route_handler`.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/360/commits/0fffa207f737387709c6aed808c410d40ec35a9b
Done

I could remove this function from route_handler.

Reply:
Nice